File: Homework1VB/Homework1VB/Form1.vb
Public Class Form1

    Private Sub Label1_Click(sender As Object, e As EventArgs) Handles Label1.Click

    End Sub

    Private Sub Label2_Click(sender As Object, e As EventArgs) Handles Label2.Click

    End Sub

    Private Sub btn1_Click(sender As Object, e As EventArgs) Handles btn1.Click
        Label2.Text = "Well done, that's a string text"
    End Sub

    Private Sub btn2_Click(sender As Object, e As EventArgs) Handles btn2.Click
        Label2.Text = String.Empty
    End Sub

    Private Sub btn3_Click(sender As Object, e As EventArgs) Handles btn3.Click
        Application.Exit()
    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Label1.Text = "Press to see a string"
    End Sub
End Class

File: Homework1C/Homework1C/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Homework1C
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            label1.Text = "Press to see a string";
        }

        private void btn1_Click(object sender, EventArgs e)
        {
            label2.Text = "Well done, that's a string text";
        }

        private void btn2_Click(object sender, EventArgs e)
        {
            label2.Text = string.Empty;
        }

        private void btn3_Click(object sender, EventArgs e)
        {
            Application.Exit();
        }
    }
}
